files/zh-cn/_redirects.txt

```diff
@@ -55,6 +55,7 @@
 /zh-CN/docs/CSS/Understanding_z-index/Stacking_without_z-index	/zh-CN/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_without_z-index
 /zh-CN/docs/CSS/Understanding_z-index/The_stacking_context	/zh-CN/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context
 /zh-CN/docs/CSS/Universal_selectors	/zh-CN/docs/Web/CSS/Universal_selectors
+/zh-CN/docs/CSS/Using_CSS_gradients	/zh-CN/docs/Web/CSS/CSS_images/Using_CSS_gradients
```

Comment:
Please remove this line, as I think the old link is a mistake. This file is used to redirect the old link to some existed pages we had before but removed now.

```suggestion
```

Reply:
I've found this line in the english version and so assumed that this link used to exist in our translation

https://github.com/mdn/content/blob/main/files/en-us/_redirects.txt#L572

Browsed the file history and it seems not actually.